src/utils/renderSchema.ts

```diff
@@ -67,7 +67,7 @@ export const renderSchema: Walker = function*(schema, level = 0, meta = { path:
 
   const { path } = meta;
 
-  for (const node of walk(parsedSchema)) {
+  for (const node of walk(resolvedSchema)) {
```

Comment:
What is the reason for that change?

Reply:
good point, development artifact, will revert